Templates/GitHubCommenter-NodeJS/index.js

```diff
@@ -0,0 +1,56 @@
+// Setup
+// 1) Follow the link https://<functionAppName>.scm.azurewebsites.net/DebugConsole & 
+//    Traverse to the path \site\wwwroot\<Name of the Function>\ & run npm install to install request package
+// 2) Add the GITHUB_CREDENTIALS as an app setting, Value for the app setting is a base64 encoded string in the following format
+//    "Username:Password" or "Username:PersonalAccessToken"
+//     Please follow the link https://developer.github.com/v3/oauth/ to get more information on GitHub authentication
+module.exports = function (context, payload) {
+    if (payload.action != "opened") {
+        return;
+    }
+
+    var comment = "{ \"body\": \"Thank you for your contribution, We will get to it shortly\" }";
+    var label = "[ \"bug\" ]";
```

Comment:
I'd use actual json, if you're using the request library you can use the `json` property and attach json object directly instead of json strings